plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal.hs

```diff
@@ -82,6 +86,23 @@ instance Pretty Log where
         <+> "for cursor position:"
         <+> pretty position
     LogCompletions logs -> pretty logs
+    LogCabalAdd logs -> pretty logs
+
+
+haskellFilesDescriptor :: Recorder (WithPriority Log) -> PluginId -> PluginDescriptor IdeState
```

Comment:
I added this descriptor because the `descriptor` below is related to the cabal files and works on them only. Since the `cabalAddCodeAction` is called from a haskell file it requires a separate descriptor.

I will clarify this in the documentation, thank you.